integrations/cdi/jpa-cdi/src/main/java/io/helidon/integrations/cdi/jpa/JpaExtension.java

```diff
@@ -337,31 +297,31 @@ public class JpaExtension implements Extension {
     /**
      * Creates a new {@link JpaExtension}.
      *
-     * <p>Normally {@link JpaExtension} classes are created
-     * automatically and only as needed by the CDI container.  End
-     * users should have no need to create instances of this
-     * class.</p>
+     * <p>Normally {@link JpaExtension} classes are created automatically and only as needed by the CDI container.  End
+     * users should have no need to create instances of this class.</p>
      *
      * @see Extension
      */
     public JpaExtension() {
         super();
-        final String cn = JpaExtension.class.getName();
-        final String mn = "<init>";
+        String cn = JpaExtension.class.getName();
+        String mn = "<init>";
         if (LOGGER.isLoggable(Level.FINER)) {
             LOGGER.entering(cn, mn);
         }
+        this.enabled = Boolean.parseBoolean(System.getProperty(this.getClass().getName() + ".enabled", "true"));
+        if (LOGGER.isLoggable(Level.FINE) && !this.enabled) {
+            LOGGER.logp(Level.FINE, cn, mn, "{0} is not enabled", this.getClass().getName());
```

Comment:
Tiny question... By default, this extension is enabled. Is there any point to changing "is not enabled" to "has been disabled" or something else like that (maybe even citing the property name the user has set??) to reinforce that it's disabled because the user did something?

Reply:
I like it. Stand by.

Reply:
I've made the suggested change. When the new machinery goes in, I'll probably change the message again since the extension will then be disabled by default.